### ISSUE TITLE
TBminer 2.01 (windows):  Ethash / Etchash not working on RX5700 (or RX580)

### ISSUE THREAD

**Issue by UselessGuru** [sp-hash/TeamBlackMiner#388]
`TeamBlackMiner-v2.01\TBMiner.exe --algo etchash --hostname eu.mining.prohashing.com --wallet *****************--port 3357 --server-passwd a=etchash,e=off,n=********,o=***********--api --api-version 1.4 --api-port 4001 --cl-devices [0]`

All etc / ethash & dual miners show the same error:

```
00:00:00 [2023-10-01 22:25:35.688]
00:00:00 [2023-10-01 22:25:35.750] OpenCL driver version: 537.42
00:00:00 [2023-10-01 22:25:35.781] Cuda driver version: 12.2
00:00:00 [2023-10-01 22:25:35.781] Cuda runtime API version: 12.0
00:00:00 [2023-10-01 22:25:35.781]
00:00:00 [2023-10-01 22:25:35.781]
00:00:00 [2023-10-01 22:25:35.781]
00:00:00 [2023-10-01 22:25:35.781] Welcome to Team Black Miner 2.01
00:00:00 [2023-10-01 22:25:35.781]
00:00:00 [2023-10-01 22:25:35.781]
00:00:00 [2023-10-01 22:25:35.781] GPU0 gfx1010:xnack- 8GB added to miner as OpenCL device
00:00:00 [2023-10-01 22:25:35.781] Trying eu.mining.prohashing.com:3357 (etchash)
00:00:00 [2023-10-01 22:25:35.781] Api server started at 127.0.0.1:4001
00:00:00 [2023-10-01 22:25:35.781] GPU0 Started CL work thread
00:00:00 [2023-10-01 22:25:35.875] eu.mining.prohashing.com: Pool responded successfully to subscribe
00:00:00 [2023-10-01 22:25:35.875] eu.mining.prohashing.com: Difficulty is set to 0.500000
00:00:00 [2023-10-01 22:25:35.953] eu.mining.prohashing.com: Pool responded successfully to authorize
00:00:00 [2023-10-01 22:25:40.198] GPU0 Starting generating etchash dag (Epoch: 307)
00:00:00 [2023-10-01 22:25:48.103] GPU0 DAG generated in 7.91 seconds (3563519kb)
00:00:00 [2023-10-01 22:25:48.103] GPU0 XIntensity set to Dynamic
```

Then back to prompt without any error message.
2.00 is working ok in the same computer:

```
00:00:00 [2023-10-01 22:23:04.030]
00:00:00 [2023-10-01 22:23:04.087] OpenCL driver version: 537.42
00:00:00 [2023-10-01 22:23:04.118] Cuda driver version: 12.2
00:00:00 [2023-10-01 22:23:04.118] Cuda runtime API version: 12.0
00:00:00 [2023-10-01 22:23:04.118]
00:00:00 [2023-10-01 22:23:04.118]
00:00:00 [2023-10-01 22:23:04.118]
00:00:00 [2023-10-01 22:23:04.118] Welcome to Team Black Miner 2.00
00:00:00 [2023-10-01 22:23:04.118]
00:00:00 [2023-10-01 22:23:04.118]
00:00:00 [2023-10-01 22:23:04.118] GPU0 gfx1010:xnack- 8GB added to miner as OpenCL device
00:00:00 [2023-10-01 22:23:04.118] Trying eu.mining.prohashing.com:3357 (etchash)
00:00:00 [2023-10-01 22:23:04.118] Api server started at 127.0.0.1:4001
00:00:00 [2023-10-01 22:23:04.118] GPU0 Started CL work thread
00:00:00 [2023-10-01 22:23:04.197] eu.mining.prohashing.com: Pool responded successfully to subscribe
00:00:00 [2023-10-01 22:23:04.197] eu.mining.prohashing.com: Difficulty is set to 0.500000
00:00:00 [2023-10-01 22:23:04.275] eu.mining.prohashing.com: Pool responded successfully to authorize
00:00:00 [2023-10-01 22:23:07.792] GPU0 Starting generating etchash dag (Epoch: 307)
00:00:00 [2023-10-01 22:23:15.545] GPU0 DAG generated in 7.77 seconds (3563519kb)
00:00:00 [2023-10-01 22:23:15.545] GPU0 XIntensity set to Dynamic
00:00:00 [2023-10-01 22:23:15.577] GPU0 Fetching etchash dag (Epoch: 307)
00:00:00 [2023-10-01 22:23:15.577] GPU0 XIntensity set to Dynamic
00:00:00 [2023-10-01 22:23:16.124] GPU0 Dag buffer verified ok!
00:00:00 [2023-10-01 22:23:34.824] Job a0d4d0a5
00:00:00 [2023-10-01 22:23:42.557] Job a0d4d10b
00:00:00 [2023-10-01 22:23:45.013] ETCHASH: GPU0 Accepted (1/1/0) Reply: 174ms
00:00:00 [2023-10-01 22:23:47.660] Job a0d4d1b7
00:00:00 [2023-10-01 22:23:55.585] Job a0d4d200
00:00:01 [2023-10-01 22:24:04.600] prohashing.com (etchash)    PING: 103ms    DIFFICULTY: 0.500    EPOCH: 307
00:00:01 [2023-10-01 22:24:04.600]
00:00:01 [2023-10-01 22:24:04.600] ID      BOARD    TYPE    KERN    XINT    TEMP    FAN     CORE    MEM     WATT
00:00:01 [2023-10-01 22:24:04.600] GPU0    5700XT   OpenCL  0       0       0/0     46      2040    1840    0
00:00:01 [2023-10-01 22:24:04.600]                                                                          0
00:00:01 [2023-10-01 22:24:04.600] ID      BOARD    HASHRATE/W    HASHRATE      AVERAGE       SHARES       RATE
00:00:01 [2023-10-01 22:24:04.600] GPU0    5700XT   0.00 H/W      0.00 H/s      57.36 MH/s    1/0/0        100.00%
00:00:01 [2023-10-01 22:24:04.600]                  0.00 H/W      0.00 H/s      57.36 MH/s    1/0/0        100.00%
00:00:01 [2023-10-01 22:24:04.600]
00:00:01 [2023-10-01 22:24:04.600] SHARES PER MINUTE: 1.00                              POOL HASHRATE: 35.79 MH/s
00:00:01 [2023-10-01 22:24:04.600]
00:00:01 [2023-10-01 22:24:15.391] Job a0d4d2a6
00:00:01 [2023-10-01 22:24:16.515] Job a0d4d2a9
00:00:01 [2023-10-01 22:24:22.875] ETCHASH: GPU0 Accepted (2/2/0) Reply: 172ms
```

**Comment by UselessGuru** [comment]
Any news?

**Comment by sp-hash** [comment]
fixed in v2.02

**Comment by UselessGuru** [comment]
I'd appreciate that you only close issues AFTER a fix has been confirmed by the creator. This makes testing much easier.
Thank you.
